File: testdata/java/array_initializer_in_assignment_ballerina_parser_error_handler_case.java
class Test {
    void test(ParserRuleContext parentCtx) {
        ParserRuleContext[] alternatives = null;
        switch (parentCtx) {
            case ARG_LIST:
                alternatives = new ParserRuleContext[] { ParserRuleContext.COMMA,
                        ParserRuleContext.BINARY_OPERATOR,
                        ParserRuleContext.ARG_LIST_END };
                break;
        }
    }
}

File: testdata/go/array_initializer_in_assignment_ballerina_parser_error_handler_case.go
package converted

type test struct {
}

func newTest() test {
	this := test{}
	return this
}

func (this *test) test(parentCtx ParserRuleContext) {
	// migrated from array_initializer_in_assignment_ballerina_parser_error_handler_case.java:2:5
	alternatives := nil
	switch parentCtx {
	case ARG_LIST:
		alternatives = []ParserRuleContext{ParserRuleContext_COMMA, ParserRuleContext_BINARY_OPERATOR, ParserRuleContext_ARG_LIST_END}
		break
	}
}
